README.md

```diff
@@ -46,3 +46,23 @@ provider "maas" {
 ### Resources and Data Sources Configuration
 
 The [docs](/docs) section contains details about each supported Terraform resource and data source.
+
+### Release process
+
+1. Checkout master and create a new branch as `release-vX.X.X`
+   ```bash
+   git checkout master  
+   git checkout -b release-vX.X.X
+   git push -u origin release-vX.X.X
+   ```
```

Comment:
```suggestion
1. Create a new branch from `master` as `release-vX.X.X`
   ```bash
   git branch release-vX.X.X master
   git push -u origin release-vX.X.X
   ```

```

Reply:
Rephrased the sentence as suggested. Thanks

Reply:
You missed the changes to the git commands?

I supplied different commands which avoid you checking out two branches (master, then the release branch) when all you need to do is create the branch and push it to GitHub.

Your version will leave your local checkout on the release which is an unlikely location to be wanting to make changes :shrug:

Reply:
Ok, I will update them. 
It is just usually how I create a new branch, off from a base. 
`git checkout -b` is for creating a new branch.